### ISSUE TITLE
Multi-monitor window spawn logic

### ISSUE THREAD

**Issue by humky** [bbidulock/adwm#21]
When I place my cursor to the second monitor and spawn a new client I expect something like bspwm's behavior, that is the window spawns at that monitor where the cursor currently is, and all further actions are sent to that monitor. In adwm the window spawns on the first monitor, I couldn't find how to pre-select a monitor. Also when I use `A + num` to get to a different tag/workspace I get seemingly random behavior `A + 2` can put windows on the second monitor, or on the first one, regardless of the pointer position, can you explain how exactly the multi-monitor logic suppose to work?

**Comment by bbidulock** [comment]
Yes, that's the way it is supposed to work: i.e. the active monitor is the monitor with the the cursor, however, there are also keyboard shortcuts for switching the active monitor and the active monitor is always the monitor that has the keyboard focus when the keyboard is focused.  The problem here is that I haven't really tested most of it and it has errors.  I lacked a good multi-monitor setup on which to test and could not get a xinerama/xrandr simulator to work well enough for testing.  Another problem I had was that most of the XDE gtk tools that were developed to work well with ADWM on a single monitor do not work at all on multiple monitors (GTK's fault in this case).  I used libwnck for most of them and it doesn't understand monitors.  Also, freedesktop.org NetWM/EWMH desktop environment and properties do not handle multiple monitors at all.  Of course the way that DWM and BSPWM and others SEPCTWM, RATPOISON get around this is to not support NetWM/EWMH at all.

So, yes, multi-monitor support ain't all that great yet.  It will once I get a couple.
